mathesar_ui/src/systems/table-view/row/RowCell.svelte

```diff
@@ -167,9 +171,7 @@
       {value}
       {isProcessing}
       {canViewLinkedEntities}
-      recordSummary={$recordSummaries
-        .get(String(column.id))
-        ?.get(String(value))}
+      recordSummary={recordName}
```

Comment:
> extract up into a common recordSummary variable that gets used in both places.

Throws lint error hence making this change 
`  175:36  error  'recordSummary' is already declared in the upper scope on line 105 column 6  @typescript-eslint/no-shadow`

Reply:
same reason to keep the variable as `recordName` and not `recordSummary`

Reply:
I pushed 9609d37cf81a27f510888e5c66a60b346efb7b8b which fixes this problem at the lower scope instead of the higher. This way we get the preferred name across more lines of the file. And we use the `rs` name in a very small scope.